model/src/test/kotlin/licenses/ResolvedLicenseInfoTest.kt

```diff
@@ -107,6 +109,22 @@ class ResolvedLicenseInfoTest : WordSpec() {
                 effectiveLicense shouldBe mit.toSpdx()
             }
         }
+
+        "filter(licenseChoices)" should {
```

Comment:
The function is called `applyChoices`, not `filter` (also fix the commit message).